src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -17,22 +17,34 @@ public abstract class AbstractConfigurationReader
 		protected static ConfigurationKeyValue Kv(string key, string value, string origin) =>
 			new ConfigurationKeyValue(key, value, origin);
 
-
-		protected internal static LogLevel ParseLogLevel(string value)
+		private static bool TryParseLogLevel(string value, out LogLevel? level)
 		{
-			switch (value.ToLowerInvariant())
+			level = null;
+			if (string.IsNullOrEmpty(value)) return false;
+			level = DefaultLogLevel();
+			return level != null;
+
+			LogLevel? DefaultLogLevel()
 			{
-				case "trace": return LogLevel.Trace;
-				case "debug": return LogLevel.Debug;
-				case "information":
-				case "info": return LogLevel.Information;
-				case "warning": return LogLevel.Warning;
-				case "error": return LogLevel.Error;
-				case "critical": return LogLevel.Critical;
-				case "none" : return LogLevel.None;
-				default: return ConsoleLogger.DefaultLogLevel;
+				switch (value.ToLowerInvariant())
+				{
+					case "trace": return LogLevel.Trace;
+					case "debug": return LogLevel.Debug;
+					case "information":
+					case "info": return LogLevel.Information;
+					case "warning": return LogLevel.Warning;
+					case "error": return LogLevel.Error;
+					case "critical": return LogLevel.Critical;
+					case "none": return LogLevel.None;
+					default: return null;
+				}
 			}
 		}
+		protected internal static LogLevel ParseLogLevel(string value)
```

Comment:
This is only used in test code if I see it correctly. I think it'd be better to only have 1 `ParseLogLevel` method.